### PR TITLE
Add missing 1.0.0 release artifact

### LINKED ISSUE #4: Use of ES6 prevents minification with UglifyJS
The variable statements like `let` and the use of template strings [here](https://github.com/flitbit/json-ptr/blob/master/index.js#L208) are unable to be minified as tracked in https://github.com/mishoo/UglifyJS2/issues/448.

It would be nice if json-ptr in the releases directory at least was transpiled to ES5 for wider browser compatibility too. I notice you have babel in the dependencies and creating releases, but it letting the `let`, string templates, and `class` definitions through.

### PR DESCRIPTION
**Pull request by cehoffman**
Closes #4

It appears the problem was that the 1.0.0 release artifact was not saved to the repository and babel wasn setup correctly to output older compatible code.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -512,4 +512,4 @@
   } else {
     root.JsonPointer = JsonPointer;
   }
-}).call(this); // eslint-disable-line no-invalid-this
+}).call(Function('return this')());
```
Comment (flitbit) [inline index.js]: I'm curious about this change... if justified/necessary, needs `// eslint-disable-line no-eval` or some such to pass linting.

Comment (cehoffman) [inline index.js]: Sorry missed this had a linter error. Basically babel was transforming this to undefined and I didn't immediately find an cli option or alternative solution through some googling.

**Comment by cehoffman** [comment]
@flitbit, I included the lint disable line and did a slight amending to improve the history

**Comment by flitbit** [comment]
Thanks for the PR. I updated the test.html to refer to your release artifact. Works fine.

**Pull request merged**